website/docs/r/storage_mover_agent.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Storage Mover"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_mover_agent"
+description: |-
+  Manages a Storage Mover Agent.
+---
+
+# azurerm_storage_mover_agent
+
+Manages a Storage Mover Agents.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "East US"
+}
+
+resource "azurerm_storage_mover" "example" {
+  name                = "example-ssm"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_storage_mover_agent" "example" {
+  name             = "example-sa"
+  storage_mover_id = azurerm_storage_mover.example.id
+  arc_resource_id  = "${azurerm_resource_group.example.id}/providers/Microsoft.HybridCompute/machines/examples-hybridComputeName"
+  arc_vm_uuid      = "3bb2c024-eba9-4d18-9e7a-1d772fcc5fe9"
+  description      = "Example Agent Description"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Storage Mover Agent. Changing this forces a new resource to be created.
+
+* `arc_resource_id` - (Required) Specifies the fully qualified resource ID of the Hybrid Compute resource for the Storage Mover Agent. Changing this forces a new resource to be created.
```

Comment:
we don't use the phrase `Resource ID` in the documentation since it's ambiguous between an Azure Resource ID and the Terraform Resource ID

Reply:
Got it. Thanks for letting me know this.